sigal/gallery.py

```diff
@@ -179,8 +179,9 @@ def __init__(self, filename, path, settings):
         base = splitext(filename)[0]
         self.date = None
         self.src_filename = filename
-        self.filename = self.url = base + '.webm'
-        self.dst_path = join(settings['destination'], path, base + '.webm')
+        if not settings['use_orig']:
+            self.filename = self.url = base + '.webm'
+            self.dst_path = join(settings['destination'], path, base + '.webm')
```

Comment:
Maybe add a test on the extension to make sure that it is supported by the video tag (webm, mp4, ogv) ?